crypto/evp_extra/evp_asn1.c

```diff
@@ -93,10 +101,9 @@ static const EVP_PKEY_ASN1_METHOD *parse_key_type(CBS *cbs) {
   }
 
 #ifdef ENABLE_DILITHIUM
-  // if |cbs| is empty we overwrite the contents with |oid| so that we can
-  // call pub_decode(ret, &algorithm, &key) with the |algorithm| populated as |oid|.
-  // We could probably use CBS_peek_asn1_tag for this, to conditionally set |algorithm|
-  // based on if peeking at the next tag to see if there is params.
+  // if |cbs| is empty after parsing |oid| from it), we overwrite the contents
```

Comment:
```Suggestion
// if |cbs| is empty after parsing |oid| from it, we overwrite the contents
```

Reply:
added in [e419b99](https://github.com/aws/aws-lc/pull/1963/commits/e419b9944904e8e30496d5a250b696997bc7e3f0)